pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, python35Packages
+, stdenv
+, bash
+, intltool
+, glib
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, wrapGAppsHook
+}:
+
+python35Packages.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
```

Comment:
I'm able to get it to build if I put the inputs I had in `propagatedBuildInputs` in `buildInputs`, I then tried wrapping the program with the following:

```
    wrapProgram $out/bin/onboard \
        --prefix "LD_LIBRARY_PATH" : "${stdenv.lib.makeLibraryPath [
                                                        gtk3
                                                        libcanberra_gtk3
                                                        libudev
                                                        bash
                                                        hunspell
                                                        isocodes
                                                        pkgconfig
                                                        xorg.libXtst
                                                        xorg.libxkbfile
                                                        libxkbcommon
                                                        intltool
                                                        python3.pkgs.pycairo
                                                        python3.pkgs.dbus-python
                                                        python3.pkgs.pygobject3
                                                        python3.pkgs.systemd
                                                        python3.pkgs.distutils_extra
                                                        python3.pkgs.pyatspi
                                                        glib ]}"
```

Is that what you meant? When I use the wrapper I get an error saying the argument list is too long, I get the message even when I'm only using a single library in `stdenv.lib.makeLibraryPath`:

```
/nix/store/7qr5cvy0wrgdqw8c21hfd7clbphm0388-onboard-1.4.1/bin/.onboard-wrapped: line 3: /nix/store/7qr5cvy0wrgdqw8c21hfd7clbphm0388-onboard-1.4.1/bin/.onboard-wrapped: Argument list too long
```
Any thoughts?